src/unix/fs.c

```diff
@@ -736,6 +741,15 @@ static ssize_t uv__fs_sendfile_emul(uv_fs_t* req) {
   return nsent;
 }
 
+#ifdef __NR_copy_file_range
+static loff_t
+uv__fs_copy_file_range(int fd_in, loff_t *off_in, int fd_out,
+                loff_t *off_out, size_t len, unsigned int flags)
```

Comment:
Style nits:

* Arguments should line up. Libuv style is one argument per line if they don't fit on a single line.
* Star leans left, e.g. `loff_t* off_in`.